qemu/tests/live_snapshot_chain.py

```diff
@@ -168,4 +168,7 @@ def do_operate(params, key_word):
             if index != 0:
                 image = qemu_storage.QemuImg(
                     image_params, data_dir.get_data_dir(), image)
+                test.assertTrue(os.path.exists(image.image_filename),
```

Comment:
Interesting use of Avocado's (unittest's, really) assertion methods.  In theory, this should work, but I haven't found other examples in tp-qemu.  Have you actually tried how Avocado-VT and Avocado behaves?

Reply:
I tried it in this test and yes, it works well.